### DISCUSSION TITLE
Quick report while I remember

### DISCUSSION THREAD

**Discussion by spamatica** [muse-sequencer/muse#1184] [Team Posts]
Some MusE testing. I can add them as bugs later.

Marker window:
Pressing delete while the markers view is highlighted deletes a track. Expected would be that the selected marker was deleted.

Not a bug but with the marker edit dialog at the bottom, it gets a bit small. I first didn't notice the edit box. 

Arranger, time signature:
Not sure when this cropped up, might be separate from the style remake.
With Snap set to Bar the calculation is off. 
Try dragging the mouse over the arranger and observe the highlight on the ruler, it jumps around.

**Comment by kybos** [comment]
> Marker window:
> Pressing delete while the markers view is highlighted deletes a track. Expected would be that the selected marker was deleted.

Not really a bug. Pressing delete did not work in the "old" marker view either. There it did nothing, as it was a separate window, but now the marker dock is in the arranger scope so the event is passed to arranger. And there it deletes the track. 
 

> Not a bug but with the marker edit dialog at the bottom, it gets a bit small. I first didn't notice the edit box.

You mean the default width of the dock should be bigger, so the name input field gets wider?

> Arranger, time signature:
> Not sure when this cropped up, might be separate from the style remake.
> With Snap set to Bar the calculation is off.
> Try dragging the mouse over the arranger and observe the highlight on the ruler, it jumps around.

Can't reproduce, for me it behaves as expected. It jumps around, yes, because it's snapping to the bars.

**Comment by kybos** [comment]
> Arranger, time signature:
>     Not sure when this cropped up, might be separate from the style remake.
>     With Snap set to Bar the calculation is off.
>     Try dragging the mouse over the arranger and observe the highlight on the ruler, it jumps around.
> 
> Can't reproduce, for me it behaves as expected. It jumps around, yes, because it's snapping to the bars.

Possibly you describe the problem mentioned here:
https://github.com/muse-sequencer/muse/issues/874#issuecomment-712577066
But it only happens after a signature change, if I understand correctly.

**Comment by kybos** [comment]
> Marker window:
> Pressing delete while the markers view is highlighted deletes a track. Expected would be that the selected marker was deleted.

I added the necessary functionality (to all docked editors). Handling of multi-shortcuts (like Del) in the same scope is very tricky, but now it should work.

> Not a bug but with the marker edit dialog at the bottom, it gets a bit small. I first didn't notice the edit box.

I redesigned the marker dialog a little, it's hopefully clearer and more user-friendly now.

**Comment by spamatica** [comment]
Yay, didn't try it out yet but I'm happy for the progress =)

Was gonna add that I had it happen after changing to 3/4 so you are probably right it's the same issue.
